docs/organizations/using-a-coding-standard.md

```diff
@@ -12,7 +12,7 @@ The organization coding standard helps you ensure that Codacy analyzes multiple
 When you customize the tools or code patterns of a repository that follows the coding standard, Codacy warns you that the repository will stop following the coding standard and asks for your confirmation.
 
 !!! important
-    The coding standard enables or disables any tools that are using configuration files on the repositories following the coding standard, but these tools will keep using configuration files instead of the code pattern settings of the coding standard.
+    The coding standard turns tools with configuration files on and off. Tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

Comment:
The original sentence is definitely very complex! I like your simplified version, I only feel that we need to reinforce that we are referring to the configuration files of the previously mentioned tools.

```suggestion
    The coding standard turns tools with configuration files on and off. Those tool configuration files, however, take precedence over the code patterns defined on the coding standard.
```

Reply:
I agree! :+1: